src/services/base/base.service.js

```diff
@@ -561,6 +675,28 @@ module.exports.search = async (args, {req}, resource_name, collection_name) => {
     logRequest(args);
     logRequest('--------');
 
+    // add any access codes from scopes
+    const accessCodes = getAccessCodesFromScopes('read', req.user, req.authInfo && req.authInfo.scope);
+    // fail if there are no access codes
+    if (accessCodes.length === 0) {
+        let errorMessage = 'user ' + req.user + ' with scopes [' + req.authInfo.scope + '] has no access scopes';
+        throw new ForbiddenError(errorMessage);
+    }
+    // see if we have the * access code
+    else if (accessCodes.includes('*')) {
```

Comment:
Either this isn't needed or need something in the line below

Reply:
@dominathan could you explain?

Reply:
resolved it per discussion with Nathan